AndroidNativeKotlinTemplate/app/build.gradle.kts

```diff
@@ -7,6 +7,9 @@ plugins {
 
 dependencies {
     implementation("com.salesforce.mobilesdk:MobileSync:12.1.0")
+    // Uncomment when enabling log in via Salesforce UI Bridge API generated QR codes
+//    implementation("com.google.zxing:core:3.4.1")
+//    implementation("com.journeyapps:zxing-android-embedded:4.3.0")
```

Comment:
One drawback with having things to uncomment is that our automated tests can't exercise that code.
Maybe we should have it in the template by default and the comment could say: "Comment the following code if you do no plan to use QR login". Another option is to have a separate template alltogether.

Reply:
Done!